api/tests/Api/Camps/ReadCampTest.php

```diff
@@ -67,6 +67,12 @@ public function testGetSingleCampIsAllowedForGuest() {
                 'categories' => ['href' => '/categories?camp=%2Fcamps%2F'.$camp->getId()],
             ],
         ]);
+
+        $responseArray = $response->toArray();
+        $period1 = static::getFixture('period1');
+        $period2 = static::getFixture('period2');
+        $this->assertEquals("/periods/{$period2->getId()}/days", $responseArray['_embedded']['periods'][0]['_links']['days']['href']);
+        $this->assertEquals("/periods/{$period1->getId()}/days", $responseArray['_embedded']['periods'][1]['_links']['days']['href']);
```

Comment:
Is the order of these two periods in the response guaranteed? Or should you use `$this->assertEqualsCanonicalizing` here?

Reply:
Yeah, should always be sorted by period start date:
https://github.com/ecamp/ecamp3/blob/devel/api/src/Entity/Camp.php#L98

period2 starts before period1:
https://github.com/ecamp/ecamp3/blob/devel/api/fixtures/periods.yml#L10